public_site/index.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<title>Lockbook</title>
+		<link rel="icon" href="/favicon.svg" type="image/svg+xml">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;700&display=swap" rel="stylesheet">
+		<link href="fontawesome/css/all.min.css" rel="stylesheet">
+		<meta name='viewport'
+			  content='width=device-width, initial-scale=1.0, maximum-scale=1.0,
+			user-scalable=0'>
+		<link rel="stylesheet" href="styles.css">
+	</head>
+	<body>
+		<div class="container">
+			<span class="left">
+				<div class="centered">
+					<div class="content">
+						<h1 id="text">LOCKBOOK</h1>
+						<h3>The best place to store and share thoughts</h3>
+						<p>
+							Privacy shouldn’t be a compromise. That’s why we made Lockbook, a companion for recording
+							thoughts on all your devices. Record, sync, and share your notes with apps engineered to
+							feel like home on every platform. We collect no personal information and encrypt your notes
+							so even we can’t see them. Don’t take our word for it: Lockbook is 100% open-source.
```

Comment:
```suggestion
							Privacy shouldn’t be a compromise. That’s why we made Lockbook, a secure companion to record, sync, and share your thoughts with apps engineered to
							feel like home on every platform. We collect no personal information and encrypt your notes
							so even we can’t see them. Don’t take our word for it: Lockbook is 100% open-source.
```
It felt a bit redundant / awkward at certain parts. This is just a rough simplification that can probably be refined further.